examples/manifests/hw-event-proxy.yaml

```diff
@@ -37,6 +37,10 @@ spec:
           env:
             - name: HW_PLUGIN
               value: "true"
+            - name: PTP_PLUGIN
```

Comment:
since this is hw specific manifests, you should remove other plugin variables from this yaml

Reply:
removed